A bit more details on "Duplicate layers" and "Copy layers" tools (#5702)
qgis/QGIS-Documentation: docs/user_manual/introduction/qgis_gui.rst

:ref:`save modifications <sec_edit_existing_layer>` to them.
Using the same data sources, you can also:

> * :guilabel:`Duplicate` a layer, generating  within the same project
>   a copy you can modify some properties (name, styles, joins...).
>   The two layers rely on the same data source.

Reviewer: ```suggestion
* :guilabel:`Duplicate` a layer to generate a copy where you can
  modify the name, style (symbology, labels, ...), joins, ...
  The copy uses the same data source as the original.
```

* :guilabel:`Copy` and :guilabel:`Paste` layers or groups from one
  project to another as a new instance whose properties can be

>   modified independently from the original project.

Reviewer: ```suggestion
  modified independently.
```
